src/monodroid/jni/monodroid-glue-internal.hh

```diff
@@ -385,6 +385,9 @@ namespace xamarin::android::internal
 		timing_period       jit_time;
 		FILE               *jit_log;
 		MonoProfilerHandle  profiler_handle;
+
+		inline static jclass    mono_android_Runtime;
```

Comment:
What does it mean to make a static field `inline`?

Reply:
It removes the necessity to define the field in a separate source file. Without it you'd have to add somewhere the following code:

```c++
jclass MonodroidRuntime::mono_android_Runtime;
```

Just a convenience really :)